src/plugins/intel_gpu/src/plugin/ops/convolution.cpp

```diff
@@ -44,7 +44,7 @@ static void CreateConvolutionOp(ProgramBuilder& p, const std::shared_ptr<ov::int
     auto pads_end = op->get_pads_end();
     auto auto_pad = op->get_auto_pad();
 
-    if (!op->is_dynamic()) {
+    if (!op->is_dynamic() && outDims.size() >= 4) {
```

Comment:
Could you explain more about this code? The commit message said "Conv doens't extend strides and dilations for 3d output". Does it mean that "conv should not extend ..."?
I think this code itself is about to handle 3d case properly because 1d vector is used when `outDims.size() == 3`. Is there a case where this condition is true?

Reply:
- Yes it means, "conv should not extend ...".
- 1d conv has 3d input/output shape. The extension code looks like old style to handling minimum 4d tensor. 
- "Is there a case where this condition is true?" --> stride is 1d at original model, but input/output becomes 4d case at original or during optimization possibly.

Reply:
Then please update commit message accordingly. Could you check whether this code is executed? I think you can check it with the broader set of models that has 1d convolution. I think we need to remove this block if it is of no use. Also, please run jenkins testing.

Reply:
I've update the condition with !p.use_new_shape_infer(). We may delete whole 'if' code block after old shpae infer is no more used.